apps/browser/components/Workspace.tsx

```diff
@@ -27,6 +30,8 @@ function Workspace({ onUpload }: WorkspaceProps) {
 
   const workspaceLoading = workspaceUploads.length > 0;
 
+  const showFileModal = !!showFile;
```

Comment:
i think we shouldn't depend on the file being loaded into state but rather have a state for opening the modal (and passing the file) - otherwise this happens (when u close the modal u can see the "View File"):

[Screencast from 20-12-23 23:54:51.webm](https://github.com/polywrap/evo.ninja/assets/21031138/00384b0d-cd60-4f8e-abe9-32968622f415)